e2eTests/.fructose/jest-setup.js

```diff
@@ -3,46 +3,97 @@ import fructose from "@times-components/fructose/setup";
 import detox from "detox";
 import webdriverio from "webdriverio";
 import { spawn } from "child_process";
+import { Chromeless } from "chromeless";
+import path from "path";
+import portscanner from "portscanner";
+import EventEmitter from "events";
 import config from "../package";
 
+const isPortTaken = port =>
+  new Promise(resolve => {
+    portscanner.checkPortStatus(port, "127.0.0.1", (error, status) => {
```

Comment:
should we handle the error?

Reply:
I think we'll be fine without it, it's not part of fructose, just the e2eTests - I haven't had any reason to think this will be an issue yet - if you can think of one that would be helpful.